lib/utils/task-option-validator.js

```diff
@@ -0,0 +1,93 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+var di = require('di');
+var path = require('path');
+
+module.exports = taskOptionValidatorFactory;
+
+di.annotate(taskOptionValidatorFactory, new di.Provide('TaskOption.Validator'));
+di.annotate(taskOptionValidatorFactory, new di.Inject(
+    'JsonSchemaValidator',
+    'FileLoader',
+    '_',
+    'Util'
+));
+
+function taskOptionValidatorFactory(
+    JsonSchemaValidator,
+    FileLoader,
+    _,
+    util
+) {
+    var TaskSchemaFolder = path.resolve(__dirname, '../../lib/task-data/schemas');
+    var MetaSchemaFileName = 'rackhd-task-schema.json';
+    var ContextPattern = /\{\{[\s\S]*context\.[\s\S]*\}\}/;
```

Comment:
@WangWinson if you want it to be like a class static variable, you can also do:

`TaskOptionValidator.ContextPattern = <...>`

and reference it the same way, e.g.

```
> function A() {}
undefined
> A.foo = 'bar'
'bar'
> A.foo
'bar'
> var a = new A();
undefined
> a.constructor.foo  // another way to do it dynamically
'bar'
>
```